common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -43,9 +46,44 @@ def test_new_textbook_upload(self):
         And I have uploaded a PDF textbook and save the new textbook information
         Then the "View Live" link contains a link to the textbook in the LMS
         """
-        self.textbook_page.open_add_textbook_form()
-        self.textbook_page.upload_pdf_file('textbook.pdf')
-        self.textbook_page.set_input_field_value('.edit-textbook #textbook-name-input', 'book_1')
-        self.textbook_page.set_input_field_value('.edit-textbook #chapter1-name', 'chap_1')
-        self.textbook_page.click_textbook_submit_button()
-        self.assertTrue(self.textbook_page.is_view_live_link_worked())
+        self.textbook_upload_page.upload_new_textbook()
+        self.assertTrue(self.textbook_upload_page.is_view_live_link_worked())
+
+    @attr('a11y')
+    def test_textbook_page_a11y(self):
+        """
+        Uploads a new textbook
+        Runs an accessibility test on the textbook page in lms
+        """
+        self.textbook_upload_page.upload_new_textbook()
+        self.textbook_view_page.visit()
+
+        self.textbook_view_page.a11y_audit.config.set_rules({
+            'ignore': [
+                'color-contrast',
```

Comment:
You need to create a JIRA ticket for the ignored roles (and include the number here). Generally we create a ticket for each individual skipped rule. Make them AC tickets.